arcgis-ios-sdk-samples/Scenes/Get Elevation at a point/GetElevationPoint.storyboard

```diff
@@ -0,0 +1,87 @@
+                            <constraint firstItem="TbC-bb-iKU" firstAttribute="bottom" secondItem="p32-Us-wmg" secondAttribute="bottom" constant="-100" id="inh-fy-AjV"/>
+                            <constraint firstAttribute="bottom" secondItem="gI1-ae-TH7" secondAttribute="bottom" id="kLx-6e-wy0"/>
+                        </constraints>
+                        <viewLayoutGuide key="safeArea" id="p32-Us-wmg"/>
+                    </view>
+                    <connections>
+                        <outlet property="elevationPointLabel" destination="TbC-bb-iKU" id="fJi-sg-fIa"/>
+                        <outlet property="sceneView" destination="gI1-ae-TH7" id="4yL-od-R8q"/>
+                    </connections>
+                </viewController>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="Kc0-vp-knm" userLabel="First Responder" sceneMemberID="firstResponder"/>
+            </objects>
+            <point key="canvasLocation" x="125" y="-23"/>
+        </scene>
+        <!--Elevation View Controller-->
+        <scene sceneID="mbo-WU-56s">
+            <objects>
+                <viewController storyboardIdentifier="ElevationViewController" automaticallyAdjustsScrollViewInsets="NO" id="0rn-Ar-HUQ" userLabel="Elevation View Controller" customClass="ElevationViewController" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target" sceneMemberID="viewController">
+                    <view key="view" opaque="NO" contentMode="scaleToFill" id="c9Z-Hn-Ppo" userLabel="Label View">
+                        <rect key="frame" x="0.0" y="0.0" width="310" height="20"/>
+                        <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
+                        <subviews>
+                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="jNi-uF-fm7" userLabel="Elevation Label">
+                                <rect key="frame" x="5" y="5" width="310" height="10"/>
+                                <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                <color key="textColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                                <nil key="highlightedColor"/>
+                            </label>
+                        </subviews>
+                        <color key="backgroundColor" white="0.66666666666666663" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                        <constraints>
+                            <constraint firstItem="jNi-uF-fm7" firstAttribute="leading" secondItem="lS6-6U-DxF" secondAttribute="leading" constant="5" id="87s-kw-gKe"/>
```

Comment:
Instead of constraining to either the superview or the safe area with a constant, please constrain all edges of the label to the superview's margins and then adjust the super view's layout margins as desired:

![image](https://user-images.githubusercontent.com/2257493/64360587-9fe81100-cfbf-11e9-826c-74e077094cf8.png)